src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -104,7 +104,7 @@ protected function extractAttributes($object, $format = null, array $context = [
 
         do {
             foreach ($reflectionObject->getProperties() as $property) {
-                if (!$this->isAllowedAttribute($reflectionObject->getName(), $property->name)) {
+                if (!$this->isAllowedAttribute($reflectionObject->getName(), $property->name, $format, $context)) {
```

Comment:
this is a bugfix i stumbled upon. doing this against 3.4 in https://github.com/symfony/symfony/pull/31071